src/macros.rs

```diff
@@ -47,8 +47,7 @@ macro_rules! json_get {
     };
     ($client:expr, $url:expr, $expected:ty, $query:expr) => {{
         let mut url = $url.clone();
-        let pairs = $query.iter();
-        url.query_pairs_mut().extend_pairs(pairs);
+        url.query_pairs_mut().extend_pairs($query);
```

Comment:
i'll admit i don't know what i'm doing here as i'm not good on rust macros, but as `$query` is now an `IntoIterator` object the `.iter()` cast was complaining, so i removed it.

Reply:
that should be correct. The macro here is pretty brittle tho 😬  would love to improve it in the future